Wordle.Api/Wordle.Api/Controllers/LeaderBoardController.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Wordle.Api.Services;
+using Wordle.Api.Dtos;
+using Wordle.Api.Models;
+
+namespace Wordle.Api.Controllers;
+
+[ApiController]
+public class LeaderBoardController : ControllerBase
+{
+    private readonly WordleServicing wordleService;
+
+    public LeaderBoardController(WordleServicing service)
+    {
+        this.wordleService = service;
+    }
+

```

Comment:
Need to add a HttpGet and HttpPost.  The get for getting the top10scores for the leaderboard and the post for posting each score after the game ends.  You can reference my LeadboardController for an idea of what to do, but it hasn't been tested yet.

Reply:
Thank you, we still need to get that.